public/components/content-list/_content-list.scss

```diff
@@ -295,17 +341,30 @@ $textualIconSize: 22px;
         list-style: none;
         padding: 0;
         margin: 0;
+        max-height: 400px;
+        overflow-x: hidden;
+        overflow-y: scroll;
     }
 
     .column-configurator__list-item {
         border-bottom: 1px solid $c-grey-300;
-        padding: 6px 10px;
         .column-configurator__label {
+            padding: 6px 10px;
             color: $c-grey-700;
             font-weight: normal;
             @extend %fs-data-3;
             text-transform: capitalize;
             margin: 0;
+            cursor: pointer;
+            width: 100%;
+
+            input {
+                width: auto;
+            }
+
+            &:hover {
+                background-color: $c-grey-200;
+            }
```

Comment:
This creates a more accessible UX.

**Before**
![menu](https://user-images.githubusercontent.com/836140/82716790-32852900-9c91-11ea-84fe-55e798ef6a4e.gif)


**After**
![menu](https://user-images.githubusercontent.com/836140/82716766-18e3e180-9c91-11ea-9e5b-a9b1044d77c2.gif)

Reply:
This one is great! One tiny nitpick would be for the label text not tot be selectable, as visible in GIF in “Assignee”.

Reply:
> the label text not tot be selectable

Fixed in https://github.com/guardian/workflow-frontend/pull/220/commits/65825e4fa23a97298ee0baa15e1f18f6bbf777a0.